### PR TITLE
Avoid using YAML in rake release to minimize file changes

### PR DESCRIPTION
**Pull request by Fryguy**
@simaishi Please review.  Alternative to #664 and #675

### CODE REVIEW

File: lib/tasks_private/release.rake [hunk 1]
```diff
@@ -3,7 +3,6 @@ Rake::Task[:release].clear
 desc "Release a new project version"
 task :release do
   require 'pathname'
-  require 'yaml'
   require 'more_core_extensions/core_ext/hash/nested'
```
Comment (chessbyte) [inline lib/tasks_private/release.rake:6]: @Fryguy with your new approach, I think you can also remove `require 'more_core_extensions/core_ext/hash/nested'` as it was being used in `content.store_path` method

Comment (Fryguy) [reply lib/tasks_private/release.rake]: Nice.

**Review by chessbyte — APPROVED** [review]

**Review by simaishi — APPROVED** [review]

**Comment by simaishi** [comment]
Kasparov backport details:

```
$ git log -1
commit b325969a33abd3a7c7226594a2d860f16cbe76f8
Author: Satoe Imaishi <simaishi@redhat.com>
Date:   Wed Oct 28 13:33:43 2020 -0400

    Merge pull request #677 from Fryguy/fix_rake_release

    Avoid using YAML in rake release to minimize file changes

    (cherry picked from commit 49b09400bbef318017b7a59f1a25a011d9a801d4)
```

**Comment by simaishi** [comment]
Jansa backport details:

```
$ git log -1
commit ba121d5dba8f35260f32930a1564c7f60d6407d8
Author: Satoe Imaishi <simaishi@redhat.com>
Date:   Wed Oct 28 13:33:43 2020 -0400

    Merge pull request #677 from Fryguy/fix_rake_release

    Avoid using YAML in rake release to minimize file changes

    (cherry picked from commit 49b09400bbef318017b7a59f1a25a011d9a801d4)
```

**Pull request merged**